woocommerce/woocommerce-android
# [M6 Orders With Coupons] Expandable product cards in order creation - part 3

## Expandable product cards in order creation - part 3
This is part 3 of the following PRs:
* Part 1: https://github.com/woocommerce/woocommerce-android/pull/9851 
* Part 2: https://github.com/woocommerce/woocommerce-android/pull/9852
* **Part 3: https://github.com/woocommerce/woocommerce-android/pull/10018**

#### ⚠️ Do not merge

### Description

This PR adds the remaining updates to the expandable product cards, as per design specs: FTOWxedKUyDbBdOrxRfMyI-fi-14%3A9617 
* Item amount picker
* Disabling buttons during order update
* Redirecting to discount edit screen
* Showing total prices with currency symbol

### Testing instructions

1. Create a new order and add some items. 
2. Verify expandable product cards match the designs.
3. Verify the amount picker updates the order as expected.
4. Verify "+ Add discount" and "Edit discount" buttons work as expected
5. Verify "Remove product" button works as expected
6. Verify that after applying a discount the totals are displayed correctly on the product card.

### Images/gif


https://github.com/woocommerce/woocommerce-android/assets/4527432/0577f84e-4b80-4d60-8035-1607514b62af
